v4/source/webhooks.yaml

```diff
@@ -26,6 +26,12 @@
               description:
                 type: string
                 description: The description for this incoming webhook
+              post_username:
```

Comment:
See the complementing PRs, but I debated whether or not to just call this parameter `username` and `icon_url` vs. the `post_` prefixed versions here. I ended up picking the `Post` prefix to complement the `Post` in `EnablePostUsernameOverride`. Let me know if you agree!

Reply:
what about override_username and override_icon_url?

Reply:
Yes, I suppose that's another option as well. The configuration being called, "PostUsernameOverride", I had originally assumed that the entity being overridden was the "post username", until I found other code just using "override username", hence the uncertainty.

Your suggestion has the nice property of aligning with the parameter name used when posting to the webhook, making things at least consistent from the API perspective.

Let me know how you'd like to proceed and I'll make any necessary changes.

Reply:
Yeah lets make the change ;)

Reply:
Will do. Note that this will of course impact https://github.com/mattermost/mattermost-webapp/pull/514 and https://github.com/mattermost/mattermost-server/pull/8002, requiring a re-review. I'll ping all of these PRs once updated.

Reply:
@enahum, looking more closely at the documentation again, I see that the parameter names used when posting a message via the incoming webhook are actually just `username` and `icon_url`. I think I had originally prefixed with `post_` when creating the integration so as to distinguish from some future model attribute, but I suspect that's not necessary.

Would you feel comfortable with converging on just `username` and `icon_url`, dispensing with both the `post_` and `override_` prefixes?

Reply:
@jwilander thoughts? I’m thinking leaving it as override_ as we have that everywhere with the webhooks

Reply:
Note that @crspeller kindly merged my approved reviews related to this change, so if we decide to change the name of this parameter, I'll open up new PRs as such.

Reply:
Yeah I'd prefer to converge on just `username` and `icon_url`. If you don't mind opening a new server PR to match, that would be great